web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -2325,7 +2332,7 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
                     name: 'dataSchema.granularitySpec.queryGranularity',
                     label: 'Query granularity',
                     type: 'string',
-                    suggestions: ['NONE', 'MINUTE', 'HOUR', 'DAY'],
+                    suggestions: ['NONE', 'SECOND', 'MINUTE', 'HOUR', 'DAY'],
```

Comment:
This is totally unrelated to this PR, but we should really consider rebranding `NONE` as `MILLIS` or `MILLISECOND` to be like... reflective of reality.

Reply:
Agree, given how the data loader is structured (and also the spirit of it) it should be a change that is made in Druid.